### PR TITLE
fix: update constant names for enum values

### PR DESCRIPTION
**Pull request by tpmsr**
Verbosify constant names per comments in https://github.com/microsoft/ConversationLearner-Models/pull/212

### CODE REVIEW

File: src/Entity.ts [hunk 1]
```diff
@@ -56,8 +56,9 @@ export function isPrebuilt(entity: EntityBase) {
   return (entity.entityName === `builtin-${entity.entityType.toLowerCase()}`)
 }
 
-export const MAX_ENUM_VALUES = 5
-export const MAX_ENUM_VALUE_LENGTH = 10
+export const MAX_ENUM_VALUES = 5  // TODO(thpar) : Obsolete, delete after references cleaned up
```
Comment (LarsLiden) [inline src/Entity.ts]: Why did this need to still be here?

Comment (tpmsr) [reply src/Entity.ts]: Backwards compatibility; I thought we could remove it once the other packages are building against the new release of Models.  Can remove if we're ok just breaking compatibility until the others are updated, LMK.

Comment (LarsLiden) [reply src/Entity.ts]: As long as you don't update the ref in package.json on the other projects it's ok.  We frequently need to push things that require changes across projects

Comment (tpmsr) [reply src/Entity.ts]: Ok; removed

**Pull request merged**